waku/waku_relay/protocol.nim

```diff
@@ -191,9 +191,8 @@ proc validator(pubsubTopic: string, message: messages.Message): Future[Validatio
 proc isSubscribed*(w: WakuRelay, topic: PubsubTopic): bool =
   GossipSub(w).topics.hasKey(topic)
 
-iterator subscribedTopics*(w: WakuRelay): lent PubsubTopic =
-  for topic in GossipSub(w).topics.keys():
-    yield topic
+proc subscribedTopics*(w: WakuRelay): seq[PubsubTopic] =
+  return toSeq(GossipSub(w).topics.keys())
```

Comment:
Morning! Is this directly related to the purpose of the PR?

Reply:
nope, but imho there is no need to use generators for this. the amount of topics is not enough to justify it. also using poorly documented features such as lent `lent` is a bad idea, unless strictly needed.